fix(gv-option): resolve display with icons & without text (#306)
gravitee-io/gravitee-ui-components: docs/components.md

# Components

Each components can use `var() - CSS` for customisation.

## How to add an CSS variable ?

> 1. Add a new line in component documentation (ex: `@cssprop {Length} [--gv-modal--w=280px] - model width`) for declare 

Reviewer: ```suggestion
1. Add a new line in component documentation (ex: `@cssprop {Length} [--gv-modal--w=280px] - model width`) to declare 
```

> the variable css. Be careful with the {Type} and use the naming convention. 

Reviewer: ```suggestion
the CSS variable. Be careful with the {Type} and use the naming convention. 
```

2. Use the custom property in own css code.
3. **ONLY** if you want that's the component to be customizable on the portal, you **MUST** add `@theme facet` in "details" section of component documentation.  

> 4. Run `npm run generate:theme` to check the variables syntax and to rebuild `definition.json` that's define the default portal's theme.

Reviewer: ```suggestion
4. Run `npm run generate:theme` to check the variable syntax and rebuild `definition.json` defining the default portal's theme.
```
